code/modules/mob/living/carbon/xenomorph/castes/Destroyer.dm

```diff
@@ -0,0 +1,90 @@
+/datum/caste_datum/destroyer
+	caste_type = XENO_CASTE_KING
+	caste_desc = "The end of the line."
+	tier = 4
+
+	melee_damage_lower = XENO_DAMAGE_TIER_6
+	melee_damage_upper = XENO_DAMAGE_TIER_8
+	melee_vehicle_damage = XENO_DAMAGE_TIER_5
+	max_health = XENO_HEALTH_DESTROYER
+	plasma_gain = XENO_PLASMA_GAIN_TIER_3
+	plasma_max = XENO_PLASMA_TIER_10
+	xeno_explosion_resistance = XENO_EXPLOSIVE_ARMOR_TIER_7
+	armor_deflection = XENO_ARMOR_FACTOR_TIER_5
+	speed = XENO_SPEED_TIER_1
+
+	minimum_evolve_time = XENO_DESTROYER_ACQUISITION_TIME // Just for the notification
+	evolves_to = null
+	deevolves_to = null
+	evolution_allowed = FALSE
+	can_vent_crawl = FALSE
+
+	behavior_delegate_type = /datum/behavior_delegate/destroyer_base
+
+	tackle_min = 6
+	tackle_max = 10
+
+	minimap_icon = "defender"
+
+	fire_immunity = FIRE_IMMUNITY_NO_DAMAGE
+
+/mob/living/carbon/xenomorph/destroyer
```

Comment:
You could have a subtype of `rogue` and in there specify a different `icon_xeno` just so the other sprite could be spawned in but still be a King.

As far as using a different wounds.dmi you can do so by overriding the rouge's Initialize with
```
/atom/movable/vis_obj/xeno_wounds/rogue
	icon = 'icons/mob/xenos/roguedamage.dmi'

/mob/living/carbon/xenomorph/king/rogue/Initialize(mapload, mob/living/carbon/xenomorph/old_xeno, hivenumber)
	..()
	vis_contents -= wound_icon_holder
	wound_icon_holder = new /atom/movable/vis_obj/xeno_wounds/rogue(null, src)
	vis_contents += wound_icon_holder
```